src/error-page.tsx

```diff
@@ -1,12 +1,12 @@
 import { useRouteError } from "react-router-dom";
 
-interface RouteError {
+interface RouteErrorProps {
```

Comment:
I'm sorry, I misled you here. I thought these are the props of the `ErrorPage` component. If they're not assigned to components' props, then no need in `Props` addition.